php/utils.php

```diff
@@ -1865,3 +1865,27 @@ function has_stdin() {
 
 	return 1 === $streams;
 }
+
+/**
+ * Return description of WP_CLI hooks used in @when tag
+ *
+ *  @param string $hook Name of WP_CLI hook
+ *
+ * @return string|null
+ */
+
+function get_hook_description( $hook ) {
```

Comment:
Can this just be a private method on the `Help_Command` class?

Reply:
I believe this needs to be a public helper class because the handbook project doesn't use the output of `wp help` and needs similar code added there as well in order to have consistency. Was going to do a PR for that one next after this is added.